homeassistant/components/rympro/sensor.py

```diff
@@ -0,0 +1,70 @@
+"""Sensor for RymPro meters."""
+from __future__ import annotations
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import UnitOfVolume
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers import entity_registry as er
+from homeassistant.helpers.entity import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import RymProDataUpdateCoordinator
+from .const import DOMAIN
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up sensors for device."""
+    coordinator: RymProDataUpdateCoordinator = hass.data[DOMAIN][config_entry.entry_id]
+    async_add_entities(
+        [
+            RymProSensor(coordinator, meter_id, meter["read"], config_entry.entry_id)
+            for meter_id, meter in coordinator.data.items()
+        ]
+    )
+
+
+class RymProSensor(CoordinatorEntity[RymProDataUpdateCoordinator], SensorEntity):
+    """Sensor for RymPro meters."""
+
+    _attr_has_entity_name = True
+    _attr_name = "Last Read"
```

Comment:
This must be following sentence case styling to be able to set `has_entity_name`.

Reply:
Additionally, it is a bit of an odd name, it would suggest a date of when it was last read? But I guess it is water usage? Would that be a better name?

Reply:
The read here is in present tense, as in the meter read. Common usage, and how the service refers to it, but maybe `total consumption`? As opposed to some periodic consumption (e.g. daily, monthly), which I plan on adding in future PRs.